src/le.rs

```diff
@@ -178,18 +178,18 @@ fn read_key_from_array(array: &[u8; LeKey::KEY_LEN]) -> LeKey {
         let u_l = array[j + 2];
         let u_r = array[j + 3];
 
-        cw[i] = CompressedCorrectionWord {
-            s,
-            z,
-            t_l,
+        *cw_i = CompressedCorrectionWord {
             u_l,
-            t_r,
+            t_l,
             u_r,
+            t_r,
+            z,
+            s,
```

Comment:
Maybe I missed it, but did you reorder the fields of the CompressedCorrectionWord struct? 
I'm starting to be worried. Correctness matters more than Clippy warnings, I'll review the rest later.

Reply:
I believe order is irrelevant in struct initialization. See [this](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=d89c838eae357c040c50318a9c20a4de) for example.

Reply:
I only reordered the values when an object was created from that struct. However, I have did this before updating cargo. Now it does not show me any warning, so I am going to delete the commit with this changes.